front/app/components/Form/Components/Controls/SingleSelectEnumControl.tsx

```diff
@@ -0,0 +1,77 @@
+import { withJsonFormsControlProps } from '@jsonforms/react';
+import {
+  ControlProps,
+  isEnumControl,
+  RankedTester,
+  rankWith,
+} from '@jsonforms/core';
+import React, { useState } from 'react';
+import ErrorDisplay from '../ErrorDisplay';
+import { Box, IOption, Select } from '@citizenlab/cl2-component-library';
+import { FormLabel } from 'components/UI/FormComponents';
+import { getLabel, sanitizeForClassname } from 'utils/JSONFormUtils';
+import styled from 'styled-components';
+import VerificationIcon from '../VerificationIcon';
+import { getOptions, getSubtextElement } from './controlUtils';
+
+const StyledSelect = styled(Select)`
+  flex-grow: 1;
+`;
+
+const SingleSelectEnumControl = ({
+  data,
+  handleChange,
+  path,
+  errors,
+  schema,
+  uischema,
+  required,
+  id,
+  visible,
+}: ControlProps) => {
+  const [didBlur, setDidBlur] = useState(false);
+  const options = getOptions(schema, 'singleEnum');
+
+  if (!visible) {
+    return null;
+  }
+
+  return (
+    <>
+      <FormLabel
+        htmlFor={sanitizeForClassname(id)}
+        labelValue={getLabel(uischema, schema, path)}
+        optional={!required}
+        subtextValue={getSubtextElement(uischema.options?.description)}
+        subtextSupportsHtml
+      />
+      <Box display="flex" flexDirection="row">
+        <StyledSelect
+          value={{
+            value: data,
+            label: 'any',
+          }} /* sad workaround waiting for PR in component library */
```

Comment:
Just made a duplicate of SingleSelectControl with a few small tweaks. I'm not sure which PR this comment is referring to - it's from the original SingleSelectControl :thinking: